### PR TITLE
[zh-cn]: create the translation of `captivePortal.onStateChanged` event

### PR DESCRIPTION
**Pull request by tianyeeT**

### CODE REVIEW

**Review by yin1999 — COMMENTED** [review]
同 #21279

File: files/zh-cn/mozilla/add-ons/webextensions/api/captiveportal/onstatechanged/index.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: onStateChanged
+slug: Mozilla/Add-ons/WebExtensions/API/captivePortal/onStateChanged
+l10n:
+  sourceCommit: 43e3ff826b7b755b05986c99ada75635c01c187c
+---
+
+当门户状态发生变化时触发。
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/captiveportal/onstatechanged/index.md:8]:
```suggestion
当强制门户状态发生变化时触发。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/captiveportal/onstatechanged/index.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+## addListener 语法
+
+### 参数
+
+- `listener`
+
+  - : 当此事件发生时调用的函数。函数被传入此参数：
+
+    - `details`
+      - : `string`。门户状态，可能是 `unknown`、`not_captive`、`unlocked_portal` 或 `locked_portal` 之一。
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/captiveportal/onstatechanged/index.md:36]:
```suggestion
      - : `string`。强制门户状态，可能是 `unknown`、`not_captive`、`unlocked_portal` 或 `locked_portal` 之一。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/captiveportal/onstatechanged/index.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+## 示例
+
+处理门户状态变化：
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/captiveportal/onstatechanged/index.md:40]:
```suggestion
处理强制门户状态变化：
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/captiveportal/onstatechanged/index.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+```js
+function handlePortalStatus(portalstatusInfo) {
+  console.log(`门户状态现在是：${portalstatusInfo.details}`);
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/captiveportal/onstatechanged/index.md:44]:
```suggestion
  console.log(`强制门户状态现在是：${portalstatusInfo.details}`);
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/captiveportal/onstatechanged/index.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+## addListener 语法
+
+### 参数
+
+- `listener`
+
+  - : 当此事件发生时调用的函数。函数被传入此参数：
+
+    - `details`
+      - : `string`。强制门户状态，可能是 `unknown`、`not_captive`、`unlocked_portal` 或 `locked_portal` 之一。
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/captiveportal/onstatechanged/index.md:38]:
```suggestion
      - : `string`。强制门户状态，为 `unknown`、`not_captive`、`unlocked_portal` 或 `locked_portal` 之一。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**